arkane/mainTest.py

```diff
@@ -53,15 +54,16 @@ def setUpClass(cls):
         """A function that is run ONCE before all unit tests in this class."""
         cls.base_path = os.path.join(os.path.dirname(os.path.dirname(rmgpy.__file__)), 'examples', 'arkane')
         cls.failed = []
-        cls.example_types = ['species', 'reactions', 'networks']
+        cls.example_types = ['species', 'reactions', 'explorer', 'networks']
 
     def test_arkane_examples(self):
         for example_type in self.example_types:
             example_type_path = os.path.join(self.base_path, example_type)
-            for example in os.listdir(example_type_path):
+            for example in sorted(os.listdir(example_type_path)):
```

Comment:
Can you check whether https://github.com/ReactionMechanismGenerator/RMG-Py/pull/1565 solved this problem?

Reply:
It doesn't solve the issue